bundles/org.openhab.binding.electroluxair/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -51,6 +51,8 @@
 			<channel id="fanSpeed" typeId="fanSpeed"/>
 			<channel id="workMode" typeId="workMode"/>
 			<channel id="ionizer" typeId="ionizer"/>
+			<channel id="uiLight" typeId="uiLight"/>
+			<channel id="safetyLock" typeId="safetyLock"/>
```

Comment:
Please add update instructions to get the new channels added automatically to managed things. See https://next.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types

Reply:
Fixed!